3581/2/env.sample

```diff
@@ -0,0 +1 @@
+TOKEN=844169864:AAHRq2KPWcpB10Nir
```

Comment:
Add `env` file to `gitignore`, nobody should see it

Reply:
This is an incorrect token. This file is created for example. Correct token is stored in a file .env.